jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6256,47 +6259,172 @@
+									"tests[\"Assert response body\"] = jsonData.applicationResponseContent.name === postman.getEnvironmentVariable(\"JDKUpgradeGroup\");",
+									"tests[\"Successful POST request\"] = responseCode.code === 201;",
+									"",
+									""
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/groups",
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{{JDKUpgradeGroup}}"
+						},
+						"description": "create group for JDKUpgradeGroup . This group will be used for JDK upgrade testing."
+					},
+					"response": []
 				},
 				{
-					"name": "upload setenv.bat template",
+					"name": "create jvm for JDK Upgrade",
 					"event": [
 						{
 							"listen": "test",
 							"script": {
 								"type": "text/javascript",
 								"exec": [
-									"tests[\"Status code is 201\"] = responseCode.code === 201;"
+									"tests[\"Status code is 201\"] = responseCode.code === 201;",
+									"var jsonData = JSON.parse(responseBody);",
+									"postman.setEnvironmentVariable(\"JDKUpgradeJvmId\", jsonData.applicationResponseContent.id.id);",
+									"tests[\"Assert response body\"] = jsonData.applicationResponseContent.jvmName === postman.getEnvironmentVariable(\"JDKUpgradeJvmName\");"
 								]
 							}
 						}
 					],
 					"request": {
-						"url": "{{url}}/v1.0/resources/template/setenv.bat.tpl",
+						"url": "{{url}}/v1.0/jvms",
 						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"jvmName\":\"{{JDKUpgradeJvmName}}\",\"groupIds\":[{\"id\":10,\"groupId\":{{JDKUpgradeGroupId}}}],\"hostName\":\"{{host}}\",\"statusPath\":\"/tomcat-power.gif\",\"systemProperties\":\"\",\"httpPort\":\"10130\",\"httpsPort\":\"10131\",\"redirectPort\":\"10132\",\"shutdownPort\":\"10133\",\"ajpPort\":\"10134\",\"userName\":\"\",\"encryptedPassword\":\"\",\"jdkMediaId\":{{JdkMediaId}},\"tomcatMediaId\":{{TomcatMediaId}}}"
+						},
+						"description": "Creating a new JVM for JDK upgrade testing."
+					},
+					"response": []
+				},
+				
+				{
+					"name": "upgrade JDK- Negative Test case",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 500\"] = responseCode.code === 500;",
+											"",
+									"var jsonData = JSON.parse(responseBody);",
+									"tests[\"Assert 500 error\"] = jsonData.status === 500;",
+
+									"tests[\"Assert No setenv.bat error\"] = jsonData.message === \"The remote JVM ::Postman-test-jvm-jdk-upgrade must be in STOPPED state before attempting to upgrade the JDK of the JVM\";"
```

Comment:
Use the environment variable for the JVM name here

Reply:
Also we need to fix the formatting of the message. Don't use 2 colons here, you can just use the JVM name. This will be a fix here and in the Java code base.